tests/features/joinup_core/page_access.feature

```diff
@@ -15,3 +15,8 @@ Feature:
     And I should see the text "The page you are looking for does not exist; It may have been moved, or removed altogether. You might want to:"
     And I should see the link "search function"
     And I should see the link "home page"
+
+  Scenario: Legal notice page is accessible by anyone.
+    When I am not logged in
+    And I visit "/joinup/legal-notice"
+    Then I should not see the heading "Page not found"
```

Comment:
Lol this is a bit of a poor test :D

Anyway, if you think it is enough to just test access this should be moved to `authentication.feature` and you should also add tests for authenticated users. Now you only know that anonymous users can see it, it might not be accessible by authenticated users.